src/App.svelte

```diff
@@ -6,16 +6,18 @@
     //import * as json from './mls.json' // imports the json file and directly loads it into the variable json.
     import JSONContentParser from "./SearchComponents/JSONContentParser.svelte"
     import {json} from './store.js'
-
+    function handleClick() {
+        alert('no more alerts')
+    }
```

Comment:
Where is this function used and what for?

Reply:
Deleted